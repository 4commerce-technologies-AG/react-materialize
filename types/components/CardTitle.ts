import * as React from 'react';

declare namespace CardTitle {
}

declare class CardTitle extends React.Component {}

export = CardTitle;
