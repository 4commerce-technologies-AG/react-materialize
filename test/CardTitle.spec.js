import React from 'react';
import { shallow } from 'enzyme';
import { assert } from 'chai';
import CardTitle from '../src/CardTitle';

let wrapper = shallow(<CardTitle image="">I am a very simple card</CardTitle>);

describe('<CardTitle />', () => {
  test('renders', () => {
    expect(wrapper).toMatchSnapshot();
  });

  wrapper = shallow(
    <CardTitle image="" reveal>
      I am a very simple card
    </CardTitle>
  );

  test('requires an image prop', () => {
    assert(wrapper.find('img.activator').length, 'with a truthy value');
  });
});
