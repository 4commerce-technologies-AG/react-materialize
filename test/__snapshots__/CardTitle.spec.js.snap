// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardTitle /> renders 1`] = `
<div
  className="card-image"
>
  <img
    className="activator"
    src=""
  />
  <span
    className="card-title"
  >
    I am a very simple card
  </span>
</div>
`;
